config/locales/activerecord.fr.yml

```
@@ -50,6 +50,9 @@ fr:
 
         operational_acceptance_done: J’atteste avoir réalisé une recette fonctionnelle et qualifié mon téléservice.
 
+        volumetrie_appels_par_minute: Quelle limitation de débit souhaitez- vous pour votre téléservice ?
+        volumetrie_justification: Pour quels motifs souhaitez-vous un débit plus grand que le minimum ?
```

C'est très étrange de décrire un attribute par une question. Je pense que tu confonds attribut et label (ce dernier doit être customiser dans l'autre fichier de locale).

```suggestion
        volumetrie_appels_par_minute: Volumétrie souhaitée
        volumetrie_justification: Justification de la volumétrie demandé
```

---

Surtout que si tu as des erreurs sur ces champs ça ve te faire `Quelle limitation de débit souhaitez- vous pour votre téléservice ? n'est pas inclut dans la liste`, ce qui est étrange.